packages/x-components/src/design-system/variables.scss

```diff
@@ -1,36 +1,36 @@
 $colors: (
-  'lead': #36515b,
-  'auxiliary': #667981,
-  'neutral-10': #1a1a1a,
-  'neutral-35': #595959,
-  'neutral-70': #b3b3b3,
-  'neutral-95': #f2f2f2,
-  'neutral-100': #ffffff,
-  'accent': #b90276,
-  'enable': #00705c,
-  'disable': #e11f26,
+  'lead': theme('colors.primary.50'),
+  'auxiliary': theme('colors.secondary.50'),
+  'neutral-10': theme('colors.neutral.90'),
+  'neutral-35': theme('colors.neutral.75'),
+  'neutral-70': theme('colors.neutral.50'),
+  'neutral-95': theme('colors.neutral.25'),
+  'neutral-100': theme('colors.neutral.0'),
+  'accent': theme('colors.accent.50'),
+  'enable': theme('colors.success.50'),
+  'disable': theme('colors.error.50'),
   'transparent': transparent
 );
 
 $sizes: (
-  '01': 2px,
-  '02': 4px,
-  '03': 8px,
-  '04': 12px,
-  '05': 16px,
-  '06': 24px,
-  '07': 32px,
-  '08': 40px,
-  '09': 48px,
-  '10': 56px,
-  '11': 64px,
-  '12': 72px,
-  '13': 80px,
-  '14': 96px,
-  '15': 112px,
-  '16': 144px,
-  '17': 176px,
-  '18': 208px,
-  '19': 240px,
-  '20': 272px
+  '01': theme('spacing.2') + 'px',
```

Comment:
would make sense to extract the unit to a variable and do something like `'01': #{theme('spacing.2')}#{$unit}` instead of concatenate the 'px' string?

Reply:
I used your approach but found out we get the same effect using `px` directly, without variable.